### PR TITLE
UI Feedbacks third pass

### PR DESCRIPTION
**Pull request by Tabrisrp**
Factorize notice HTML output

### CODE REVIEW

File: inc/admin/ui/notices.php [hunk 1]
```diff
@@ -10,77 +10,57 @@ function rocket_bad_deactivations() {
...
-				$plugin_file = 'wp-rocket/wp-rocket.php';
-				$rocket_nonce = wp_create_nonce( 'force_deactivation' );
+		}
 
-				echo '<p><a href="' . wp_nonce_url( 'plugins.php?action=deactivate&amp;rocket_nonce=' . $rocket_nonce . '&amp;plugin=' . $plugin_file . '&amp;plugin_status=' . $status . '&amp;paged=' . $page . '&amp;s=' . $s, 'deactivate-plugin_' . $plugin_file ) . '">' . __( 'You can still force deactivation by clicking here.', 'rocket' ) . '</a></p>';
-			}
-			?>
-		</div>
+		if ( count( $errors ) ) {
+			$errors = array_map( 'sprintf', $errors );
```
Comment (Screenfeed) [inline inc/admin/ui/notices.php]: I can't see the purpose of this 🤔

**Comment by Tabrisrp** [comment]
what would you recommend for the first point? Having the `<p>` tags individually?

**Comment by Screenfeed** [comment]
I usually do that kind of thing:
```php
$tag = 0 !== strpos( $message, '<p' ) && 0 !== strpos( $message, '<ul' );

echo ( $tag ? '<p>' : '' ) . $message . ( $tag ? '</p>' : '' );
```

**Comment by Screenfeed** [comment]
The code in my previous comment allows you not to add the `<p>` tags everywhere. Did add them for consistency/readability?

**Comment by Tabrisrp** [comment]
It's to ensure correct formatting in those cases

**Pull request merged**